libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -28,6 +30,7 @@ table.calendar-table {
     padding: 0;
     line-height: 2em;
     height: 0;
```

Comment:
I know you didn't apply this, but it's an odd way to (not) control the height of table cells. Some housekeeping/making-styles-sane-again™️ would be appreciated 😎

Reply:
Removed in 6e10812f252fb47ce917ea413a5d2d1a30722f67.